lib/matplotlib/streamplot.py

```diff
@@ -193,26 +199,37 @@ def streamplot(axes, x, y, u, v, density=1, linewidth=None, color=None,
         points = np.transpose([tx, ty]).reshape(-1, 1, 2)
         streamlines.extend(np.hstack([points[:-1], points[1:]]))
 
-        # Add arrows half way along each trajectory.
+        if use_multicolor_lines:
+            color_values = interpgrid(color, tgx, tgy)[:-1]
+            line_colors.append(color_values)
+
+        if numarrows <= -1:
+            raise ValueError("The value of numarrows must be"
+                             "greater than -1")
```

Comment:
```suggestion
        if numarrows < 0:
            raise ValueError("The value of numarrows must not"
                             "be negative")
```
Technically the same, but writing like this is more logical.